static/src/js/middleware/metrics/events.js

```diff
@@ -177,6 +177,42 @@ export const map = (action) => {
   })
 }
 
+/**
+* Pushes an add collection to project event on the dataLayer.
+* This event is fired when a user adds a collection to their project
+* @param {Object} action - The action.
+*/
+export const addCollectionProject = (action) => {
+  const { payload } = action
+  const {
+    type,
+    collectionConceptId
+  } = payload
+  dataLayer.push({
```

Comment:
Make this look this like:
` collectionConceptId:,
            view: '',
            page: `
           
For the event structure